<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Parent_Light_to_Rig_groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Parent Light to Rig</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Parent_Key_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent Key_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Parent_Fill_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent Fill_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Parent_Rim_L_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent Rim_L_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Parent_Rim_R_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent Rim_R_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Parent_Bounce_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent Bounce_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Parent_All_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Parent All Lights</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Lights</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Create_Key_Light_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Create Key_Light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Create_Fill_Light_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Create Fill_Light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Create_Rim_L_Light_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Create Rim_L_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Create_Rim_R_Light_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Create Rim_R_Lights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Create_Bounce_Light_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Create Bounce_Light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Group_All_Lights_pushButton">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Group All Lights</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
